### PR TITLE
Fixes Emitters: Quickmerge Edition

### PR DESCRIPTION
**Pull request by Robustin**
Meme of the Day: Assuming maintainer review requests aren't going to break the code.

### CODE REVIEW

**Comment by AnturK** [comment]
More like applying changes without testing of forethought.

File: code/modules/power/singularity/emitter.dm [hunk 1]
```diff
@@ -199,8 +199,6 @@
 
 /obj/machinery/power/emitter/proc/fire_beam(atom/targeted_atom, mob/user)
 	var/turf/targets_from = get_turf(src)
-	if(targeted_atom == user|| targeted_atom == targets_from)
```
Comment (AnturK) [inline code/modules/power/singularity/emitter.dm]: Awful fix, do if(targeted_atom && (...)) instead

**Comment by Robustin** [comment]
I just know that after a month I'm no longer going to be able to testing the 75th maintainer comment. So when a comment tells me to explicitly move a line somewhere I'll do it, if that change breaks emitter code, I just feel like I shouldn't bear full responsibility for it.

File: code/modules/power/singularity/emitter.dm (hunk 1 above)
Comment (Robustin) [inline code/modules/power/singularity/emitter.dm]: Apparently my commit wasn't recognizing as changing this line but uh... yea it's been moved now.

**Pull request merged**